bdio2/src/main/java/com/blackducksoftware/bdio2/Bdio.java

```diff
@@ -1054,6 +1047,45 @@ public static Context forSpecVersion(String specVersion) {
         }
     }
 
+    // TODO additional scan types will be added in the future releases
+    public enum ScanType {
+
+        /**
+         * A Package manager scan.
+         */
+        PACKAGE_MANAGER("PACKAGE_MANAGER"),
+        /**
+         * A File System scan.
+         */
+        FILE_SYSTEM("FILE_SYSTEM");
```

Comment:
What is the externally facing label for signature scans?  I think we commonly refer to it as signature scans, then I'd recommend enumerating this as 'SIGNATURE' rather than 'FILE_SYSTEM'.

Reply:
Sure if that is the case, will change it to SIGNATURE.

Reply:
I think we internally refer to it as a 'SIGNATURE' scan but actually, we are doing a 'FILE_SYSTEM' scan, right? So, if. we want to refer to it with a name that is internally used we can use 'SIGNATURE'.

Reply:
That's true, but, pedantically, a binary scan is technically also a file system scan.

Reply:
yes, that sounds fair. Let's go ahead with 'SIGNATURE' then.

Reply:
Please coordinate with @vigneshd-snps on this as he is working it on another ticket too!

Reply:
Change scan type FILE_SYSTEM to SIGNATURE